ambry-api/src/main/java/com.github.ambry/config/CloudConfig.java

```diff
@@ -14,7 +14,38 @@
 package com.github.ambry.config;
 
 /**
- * The class for cloud related configurations.
+ * The configs for cloud related configurations.
  */
 public class CloudConfig {
+
+  @Config("vcr.cluster.zk.connect.string")
+  @Default("")
+  public final String vcrClusterZkConnectString;
+
+  /**
+   * The name of the associated vcr cluster for this node.
+   */
+  @Config("vcr.cluster.name")
+  public final String vcrClusterName;
+
+  /**
+   * The ssl port number associated with this node.
+   */
+  @Config("vcr.ssl.port")
+  @Default("null")
+  public final Integer vcrSslPort;
+
+  /**
+   * The ssl port number associated with this node.
+   */
+  @Config("vcr.replica.mount.path.prefix")
+  @Default("/tmp/vcr/")
+  public final String vcrReplicaMountPathPrefix;
+
+  public CloudConfig(VerifiableProperties verifiableProperties) {
+    vcrClusterZkConnectString = verifiableProperties.getString("vcr.cluster.zk.connect.string", "");
+    vcrClusterName = verifiableProperties.getString("vcr.cluster.name");
+    vcrSslPort = verifiableProperties.getInteger("vcr.ssl.port", null);
```

Comment:
Remove default or replace with actual value.  (Can we use same port as ambry server?)